mockgcp/Makefile

```diff
@@ -74,10 +75,15 @@ generate-protos-from-openapi:
 	mkdir -p apis/mockgcp/cloud/bigquery/v2/
 	cd tools/gapic; go run . --proto-version=2 ../../bigquery-v2.json > ../../apis/mockgcp/cloud/bigquery/v2/api.proto
 
+	wget -O containeranalysis-v1.json https://raw.githubusercontent.com/googleapis/google-api-go-client/main/containeranalysis/v1/containeranalysis-api.json
```

Comment:
It looks like we have a proto version of this API.  The OpenAPI -> proto conversion is always trickier, so you could try using the proto version (i.e. just reference it from `./third_party/google/devtools/containeranalysis/v1`.  You likely need to add a replacement to `./fixup-third-party.sh` to rewrite `google/devtools` => `mockgcp/devtools`

Reply:
Actually, it wasn't _that_ proto (so I understand why!), but the proto looks to be available under grafeas

I made some progress here: https://github.com/GoogleCloudPlatform/k8s-config-connector/compare/master...justinsb:k8s-config-connector:mock-containeranalysisnote?expand=1

Reply:
Thanks a lot!! I was using a wrong proto so that's why none of my CRUD methods are invoked. 

Do you have any suggestions on identifying the proto used for a resources? I searched for the service name in [googleapis](https://github.com/googleapis/googleapis) and I tried code search the method name in file type `proto`. I hope the service team have this information documented somewhere...